docs/fsharp/language-reference/access-control.md

```diff
@@ -13,7 +13,7 @@ In F#, the access control specifiers `public`, `internal`, and `private` can be
 
 - `public` indicates that the entity can be accessed by all callers.
 
-- `internal` indicates that the entity can be accessed only from the same assembly.
+- `internal` indicates that the entity can be accessed only from the same [assembly](https://github.com/dotnet/docs/blob/main/docs/standard/assembly/index.md).
```

Comment:
Let's change this link to the published doc instead of the source:

```suggestion
- `internal` indicates that the entity can be accessed only from the same [assembly](../../standard/assembly/index.md).
```